src/frontend/context/fagsak/FagsakContext.tsx

```diff
@@ -50,9 +49,7 @@ const [FagsakProvider, useFagsakContext] = createUseContext(() => {
             påvirkerSystemLaster,
         })
             .then((hentetFagsak: Ressurs<IMinimalFagsak>) => {
-                if (påvirkerSystemLaster || !deepEqual(hentetFagsak, minimalFagsak)) {
-                    settMinimalFagsak(hentetFagsak);
-                }
+                settMinimalFagsak(hentetFagsak);
             })
```

Comment:
Dersom vi faktisk har gjort et eksternt kall for å hente fagsaken ser jeg ikke hvorfor vi ikke bare kan sette den.

Reply:
Fører ikke til en loop av kall når jeg tester det i localhost